web/src/routes/Events.jsx

```diff
@@ -37,6 +37,21 @@ const monthsAgo = (num) => {
   return new Date(date.getFullYear(), date.getMonth(), date.getDate()).getTime() / 1000;
 };
 
+const clipLength = (num) => {
+  let totalSeconds = num;
+  let hours = Math.floor(totalSeconds / 3600);
+  totalSeconds %= 3600;
+  let minutes = Math.floor(totalSeconds / 60);
+  let seconds = (totalSeconds % 60).toFixed(0);
+  let length=`${hours} Hours ${minutes} Minutes ${seconds} Seconds`
+  if (hours == 0 && minutes == 0) {
+    length=`${seconds} Seconds`
+  } else {
+    length=`${minutes} Minutes ${seconds} Seconds`
+  }
```

Comment:
What if we used abbreviations instead?


```suggestion
  if (hours == 0 && minutes == 0) {
    length=`${seconds}s`
  } else {
    length=`${minutes}m ${seconds}s`
  }
```

Reply:
I like that but I'm not 100% sure what that means for translations... Is m and s really international (thinking about Asian languages, etc)?

Reply:
In the final commit "Minutes/Seconds" are used (identical to the existing solution in RecordingPlaylist.jsx).
For Translation see #947 (that's what I found regarding internationalization/language resources).